packages/shared/src/components/search/SearchSourceList.tsx

```diff
@@ -31,11 +31,11 @@ export const SearchSourceList = ({
       <div className={classNames('flex flex-col', widgetClasses)}>
         <div
           className={classNames(
-            'flex justify-between items-center py-1.5 laptop:py-4 px-4 laptop:bg-transparent rounded-t-16 bg-theme-bg-secondary',
+            'flex justify-between items-center py-1.5 laptop:py-4 px-4 pb-0 laptop:bg-transparent rounded-t-16 bg-theme-bg-secondary',
             isSourcesOpen ? 'rounded-t-16' : 'rounded-16',
           )}
         >
-          <p className="font-bold typo-callout text-theme-label-tertiary laptop:text-theme-label-quaternary">
+          <p className="font-bold typo-callout text-theme-label-quaternary laptop:text-theme-label-quaternary">
```

Comment:
I think this should be on tertiary from the Sketch.

![Screenshot 2023-08-25 at 9 29 11 PM](https://github.com/dailydotdev/apps/assets/13744167/1e0c35a3-7ae6-42d6-8ee0-57a652783f91)